wasmtime/_func.py

```diff
@@ -16,6 +17,13 @@
 
 class Func:
     _func: ffi.wasmtime_func_t
+    _ty: FuncType
+    _params_n: int
+    _results_n: int
+    _params_str: list[str]
+    _results_str: list[str]
+    _results_str0: str
```

Comment:
I think this would be best stored as `Optional[str]` to avoid having a dummy value listed for empty results.